cmake/fw_zip.cmake

```diff
@@ -0,0 +1,31 @@
+
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+function(generate_dfu_zip zip_out bin_file_list deps)
```

Comment:
What will happen the day someone calls this function twice, in order to create two different zip-packages for dfu uploading.

If they have common dependency to, for example Zephyr hex target, then this will fail, as there will be two targets with identical names.